From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ismael Luceno <ismael@sourcemage.org>
Date: Sat, 22 Aug 2026 12:00:00 +0200
Subject: [PATCH] iconv: Honour ICONV_CONST

The second argument of iconv() is only const in the GNU implementation;
configure already computes ICONV_CONST for exactly this.

Upstream-Status: Pending
Signed-off-by: Ismael Luceno <ismael@sourcemage.org>
---
 packages/iconv/iconv.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/iconv/iconv.c b/packages/iconv/iconv.c
index 80c97c666ea9..e7904cc4e5d0 100644
--- a/packages/iconv/iconv.c
+++ b/packages/iconv/iconv.c
@@ -66,7 +66,7 @@ iconvWrapper (iconv_t handle, OOP readBufferOOP, int readPos,
 	      int readCount, OOP writeBufferOOP, int writeCount,
 	      OOP bytesLeftOOP)
 {
-  const char *inbuf;
+  ICONV_CONST char *inbuf;
   size_t inbytesleft;
   char *outbuf;
   size_t outbytesleft;
